src/server/products/get-products.server.ts

```diff
@@ -59,6 +60,12 @@ export async function getProducts(
       variables: product.variables,
       statusLabel: product.statusLabel,
       nutritionalValue: product.nutritionalValue,
+      available: product.available,
```

Comment:
we talked about using existing fields

Reply:
This is no longer used. I just forgot to delete it.